modules/serverless-creating-subscriptions.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+// * serverless/knative_eventing/serverless-subscriptions.adoc
+
+[id="serverless-creating-subscriptions_{context}"]
+= Creating a Subscription
+
+You can create a Subscription to connect a Service or other event sink to a channel.
+The following procedure shows how to create a Subscription object that uses the default InMemoryChannel type channel.
```

Comment:
another use of "subscription object uses a channel", not terribly wrong, but it probably be better with "connecting" or something///  (as subscription isn't really "using" a channel).

Also, in the example we're no longer explicitly mentioning the channel type, so we don't need to mention it here either. We could probably make this whole subscription section a bit more concise if we drop any mention of the InMemoryChannel or channel types altogether, and just keep the reference to serverless-channels.adoc, where they are described.